lib/Ohmbrewer_Thermostat.cpp

```diff
@@ -1,93 +1,99 @@
 #include "Ohmbrewer_Thermostat.h"
 #include "Ohmbrewer_Screen.h"
 #include "Ohmbrewer_Publisher.h"
+#include "pid.h"
 
+//Define Variables we'll be connecting to with PID
+double setPoint;
+double input;
+double output;
 
+//Define the aggressive and conservative Tuning Parameters for PID
+double aggKp=4;
+double aggKi=0.2;
+double aggKd=1;
+double consKp=1;
+double consKi=0.05;
+double consKd=0.25;
 
-/**
- * The desired target temperature. Defaults to Celsius
- * @returns The target temperature in Celsius, as a Temperature object pointer
- */
-Ohmbrewer::Temperature* Ohmbrewer::Thermostat::getTargetTemp() const {
-    return _targetTemp;
-}
-
-/**
- * Sets the target temperature
- * @param targetTemp The new target temperature in Celsius
- * @returns The time taken to run the method
- */
-const int Ohmbrewer::Thermostat::setTargetTemp(const double targetTemp) {
-    unsigned long start = millis();
-
-    _targetTemp->set(targetTemp);
+//PID windowing variables
+int windowSize = 5000;              //changeable value
+unsigned long windowStartTime;
 
-    return start - millis();
-}
+//PID set up
+PID* thermPID;
```

Comment:
thermPID should be set up kind of like what we did with Probe instead of floating around in the global space. You don't need to make a wrapper around it, but you should provide it as a member of Thermostat and provide getters, setters, and mentions in constructors/destructors.